### PR TITLE
Add RxTodo

### PR DESCRIPTION
**Pull request by devxoul**

### CODE REVIEW

File: data/items.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ sections:
       - name: Count It
         link: https://github.com/PiXeL16/CountItApp
         description: Never lose the count again. App with Apple Watch integration that lets you count anything.
+      - name: RxTodo
+        link: httpshttps://github.com/devxoul/RxTodo
```
Comment (ashfurrow) [inline data/items.yml]: Looks like a double https snuck in.

File: data/items.yml [hunk 2]
```diff
@@ -17,6 +17,9 @@ sections:
...
+        description: iOS Todo Application with RxSwift + MVVM
```
Comment (ashfurrow) [inline data/items.yml]: Could you add a `.` at the end of the sentence to keep things consistent?

**Comment by ashfurrow** [comment]
Looks good to merge, I'll fix afterward.

**Comment by ashfurrow** [comment]
Hey there! Thanks a lot for contributing! I've invited you to join the RxSwiftCommunity GitHub organization – no pressure to accept! If you'd like more information on what that means, check out our [contributor guidelines](https://github.com/RxSwiftCommunity/contributors) and feel free to reach out with any questions.

**Comment by devxoul** [comment]
@ashfurrow, It would be really fun. Thanks! :smile:

**Pull request merged**